data-hibernate-jpa/src/test/java/io/micronaut/data/hibernate/entities/EventTest.java

```diff
@@ -23,6 +23,15 @@ public class EventTest {
     @DateUpdated
     private LocalDateTime dateUpdated;
 
+    private int value;
+
+    public EventTest(int value) {
+        this.value = value;
```

Comment:
Providing an `update` without changing anything didn't make much sense and I didn't want the tests to rely on obscure Hibernate behavior (like, off the top of your head, do you know if a no-op update actually kicks off any listeners? Should it? etc.)

Reply:
Actually, Hibernate just like JPA doesn't have `update` ;) which makes things more complicated.